src/sentry/scripts/utils/locking/delete_lock.lua

```diff
@@ -0,0 +1,12 @@
+local key = KEYS[1]
+local uuid = ARGV[1]
+
+local value = redis.call('GET', key)
+if not value then
+    return redis.error_reply(string.format("No lock at key exists at key: %s", key))
+elseif value ~= uuid then
```

Comment:
Lua is the worst. I was thinking this was a regular expression test or something. Not `DOES NOT EQUAL`. :fire: